### ISSUE TITLE
Feature: Create new posts

### ISSUE THREAD

**Issue by noplanman** [Patreon/platform-documentation#29]
Are there any plans to include post creation via API?

It would be great to automate this process for release notes, when a new version of a software is released for example.

(not sure this is the correct place to suggest API features, but where else would i do this?)

**Comment by jennbriden** [comment]
I agree that this would be a substantial addition to the API. It would allow bidirectional or unidirectional sync from WordPress to Patreon. The WordPress plugin only supports syncing posts FROM Patreon.
